spec/spec_helper.rb

```diff
@@ -1,9 +1,32 @@
-# This file is generated by ModuleSync, do not edit.
 require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet-facts'
 
-# put local configuration and setup into spec_helper_local
 begin
-  require 'spec_helper_local'
+  require 'spec_helper_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_local.rb'))
 rescue LoadError => loaderror
-  puts "Could not require spec_helper_local: #{loaderror.message}"
+  warn "Could not require spec_helper_local: #{loaderror.message}"
+end
+
+include RspecPuppetFacts
+
+# rubocop:disable Style/HashSyntax
```

Comment:
This is a work around that will need to be resolved, the change will be overwritten on the next `pdk update`.

Reply:
I have raised an internal bug for tracking this and it has been added onto the backlog.
We will need to come up with some type of solution going forward.